File: Java/Clothing.java
public class Clothing extends Product {
    private char size;
    private String material;
    private char gender;

    public Clothing(char size, String material, char gender, int id, String name, String brand, String price) {
        super(id, name, brand, price);
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    // Other getters and setters
    public char getSize() {
        return size;
    }

    public void setSize(char size) {
        this.size = size;
    }

    public String getMaterial() {
        return material;
    }

    public void setMaterial(String material) {
        this.material = material;
    }

    public char getGender() {
        return gender;
    }

    public void setGender(char gender) {
        this.gender = gender;
    }
}

File: PHP/Program/Clothing.php
<?php

require_once 'Product.php';

class Clothing extends Product {
    private $size;
    private $materials;
    private $gender;

    public function __construct($size = "", $materials = "", $gender = "") {
        parent::__construct(); // Panggil konstruktor dari kelas induk
        $this->size = $size;
        $this->materials = $materials;
        $this->gender = $gender;
    }

    public function get_size() {
        return $this->size;
    }

    public function set_size($size) {
        $this->size = $size;
    }

    public function get_materials() {
        return $this->materials;
    }

    public function set_materials($materials) {
        $this->materials = $materials;
    }

    public function get_gender() {
        return $this->gender;
    }

    public function set_gender($gender) {
        $this->gender = $gender;
    }

    public function __destruct() {
        
    }
}

?>